src/contracts/StkABPTMigrator.sol

```diff
@@ -110,7 +124,7 @@ contract StkABPTMigrator is Rescuable {
         poolOutAmountMin
       );
     }
-    AggregatedStakedTokenV3(STK_ABPT_V2).stake(
+    IAggregatedStakeToken(STK_ABPT_V2).stake(
```

Comment:
I think back then we discussed and came to the conclusion that the same can happen with ABPT_V1, but eventually decided to ignore 😅 

I can add a check, but it will require 4 more balanceOf calls, idk if worth it.

Reply:
Mainly to be consistent with the rest, I would probably add it

Reply:
okay added